base/float.jl

```diff
@@ -858,6 +860,19 @@ julia> floatmin(Float32)
 julia> floatmin()
 2.2250738585072014e-308
 ```
+
+# Extended help
+
+The smallest positive number without this restriction is
+[`nextfloat`](@ref)`(zero(T))`; see [`issubnormal`](@ref) for an explanation
+of sub-normal numbers.
+
+Floating point numbers always include infinity, [`Inf`](@ref), and
+negative real infinity, `-Inf`, which is what [`typemax`](@ref)`(T)`
+and [`typemin`](@ref)`(T)`, for a floating-point type `T`, would return.
+
+The smallest finite number representable by a floating-point type `T`
+is gotten by negating `floatmax` i.e. `-`[`floatmax`](@ref)`(T)`.
```

Comment:
This is wrong. `-floatmax(T)` is a huge negative number, not small.

Reply:
I think the "lowest finite number" solves it.